internal/incident/incidents.go

```diff
@@ -25,38 +27,122 @@ var (
 )
 
 // LoadOpenIncidents loads all active (not yet closed) incidents from the database and restores all their states.
-// Returns error ony database failure.
+// Returns error on any database failure.
 func LoadOpenIncidents(ctx context.Context, db *icingadb.DB, logger *logging.Logger, runtimeConfig *config.RuntimeConfig) error {
 	logger.Info("Loading all active incidents from database")
 
-	var objectIDs []types.Binary
-	err := db.SelectContext(ctx, &objectIDs, `SELECT object_id FROM incident WHERE "recovered_at" IS NULL`)
-	if err != nil {
-		logger.Errorw("Failed to load active incidents from database", zap.Error(err))
+	g, chCtx := errgroup.WithContext(ctx)
```

Comment:
You could just set `ctx` here instead instead of introducing `chCtx`. The rest of the function is supposed to use one of these only anyways, so just avoid the confusion of having two different context variables to choose from.